vscode-extension/static/untitled.aiconfig.json

```diff
@@ -11,21 +12,21 @@
     },
     "default_model": "gpt-3.5-turbo"
   },
-  "description": "Fill out a description about your AIConfig here.",
+  "description": "Enter a description for this AIConfig",
```

Comment:
I think a good compromise between keeping it clean and informative would be to link it to our AICOnfig getting started docs